config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
If `debug`, `locale` and `featureManagerIsFullZoneProvisioningEnabled` will be configurable I think we should add all of them in the sample. It provides an ease of mind to the user when it's already there with default settings. 

Additionally, we can make the README more explicit.